packages/app/src/utils/dynamicUI/extractDynamicConfig.ts

```diff
@@ -22,12 +24,22 @@ export type MenuItem =
       };
     };
 
+export type MenuItem = {
+  name: string;
+  title: string;
+  icon: string;
+  children: MenuItem[];
```

Comment:
A MenuItem in the config doesn't support children's, or?

```suggestion
```

Reply:
This is not for the config but for the tree structure. I had one type only for config but you suggested to only have `MenuItem`.